R/merPredict.R

```diff
@@ -254,6 +267,41 @@ predictInterval <- function(merMod, newdata, which=c("full", "fixed", "random",
   ##Calculate yhat as sum of the components (fixed plus all groupling factors)
   fe.tmp <- fixef(merMod)
   vcov.tmp <- as.matrix(vcov(merMod))
+  if (fix.intercept.variance) {
+    #Assuming all random effects include intercepts.
+    intercept.variance = vcov.tmp[1,1]
+
+    groupsizes = ngrps(merMod)
+    for(j in names(groupsizes)){ #for every group of random e
+
+      groupExtraPrecision = 0
+      groupVar = (attr(VarCorr(cvoteg)[[j]],"stddev")["(Intercept)"])^2
```

Comment:
Appears that `cvoteg` should be `merMod`, no?  Is this the name of the model you tested on?